#!/usr/bin/env python
PACKAGE = "task_switch"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add("int_param", int_t, 0, "An Integer parameter", 50, 0, 100)
gen.add("controller_gain", double_t, 0, "A double parameter", .1, 0, 10)
# gen.add("str_param", str_t, 0, "A string parameter", "Hello World")
# gen.add("bool_param", bool_t, 0, "A Boolean parameter", True)


exit(gen.generate(PACKAGE, "task_switch", "pcc_params"))
